domains/nulitas.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "nulitas",
+  },
```

Comment:
You need email line. If you don't want to give your email then leave it blank and add another form of contact (like discord, twitter, reddit, mastodon etc.)